app/Http/Transformers/AccessoriesTransformer.php

```diff
@@ -51,6 +51,8 @@ public function transformAccessory(Accessory $accessory)
             'checkin' =>  false,
             'update' => Gate::allows('update', Accessory::class),
             'delete' => Gate::allows('delete', Accessory::class),
+            'clone' => (Gate::allows('create', Accessory::class) && ($accessory->created_at == 'datetime')),
```

Comment:
Do we need the `&& ($accessory->created_at == 'datetime')`? here? Not sure what it does.